src/picknik_ur_gazebo_config/waypoints/waypoints.yaml

```diff
@@ -349,7 +349,7 @@
     velocity: []
   name: Monitor View
 - description: ''
-  favorite: true
+  favorite: false
```

Comment:
unfavorited this "Extended Right" waypoint.

Reply:
how dare you